### DISCUSSION TITLE
Allowing using location

### DISCUSSION THREAD

**Discussion by jojurko** [getgauge/taiko#2656] [General]
How I can allow using location on webpage?
![image](https://user-images.githubusercontent.com/43095337/200301678-c3a4c43c-0d9c-47a8-884a-7ebb91ddcd07.png)

**Comment by marques-work** [comment]
Maybe check the startup flags in Chromium? If there isn't a flag, I suspect you might need to create a profile and manually allow permissions; then save and activate that profile each time you start chromium.

**Reply by jojurko** [reply]
thank you, I am not able to find relevant flag, or identify correct one :(  
do you mean profile on google? https://support.google.com/a/users/answer/9310144?hl=en

**Reply by marques-work** [reply]
No, I mean a chromium user profile -- basically a set of settings for a user.

This concept here: https://www.chromium.org/developers/creating-and-using-profiles/

**Comment by DCoomer** [comment]
Would https://docs.taiko.dev/api/setlocation/ work?

**Comment by jojurko** [comment]
it seems, I should better study existing api :)
https://docs.taiko.dev/api/overridepermissions/
